cluster/addons/istio/auth/istio-auth.yaml

```diff
@@ -1,5 +1,15 @@
 # GENERATED FILE. Use with Kubernetes 1.7+
 # TO UPDATE, modify files in install/kubernetes/templates and run install/updateVersion.sh
+#
+# Required for Istio as k8s addon.
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: default
+  labels:
+    istio-injection: enabled
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Will this prevent users to add other labels to default namespace?

Reply:
No, only items explicitly set in the yaml are reconciled (so you only cannot remove this label). Users are free to add anything not listed in the yaml.